scripts/skill_auditor/test_skill_auditor.py

```
@@ -0,0 +1,323 @@
+    """Test that validation passes with complete metrics."""
+    complete_metrics = {
+        "forbidden_files": [],
+        "yaml_delimiters": 2,
+        "has_name": True,
+        "has_description": True,
+        "line_count": 100,
+        "implementation_details": [],
+        "quoted_count": 5,
+        "domain_count": 5,
+        "skill_name": "test",
+        "skill_path": "/tmp/test",
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)